README.md

```diff
@@ -119,3 +119,4 @@ Testing steps:
 - ES SQL API does not support `SELECT DISTINCT`, but we can achieve the same result by `COUNT(DISTINCT colName)`
 - ES SQL API does not support `ORDER BY aggregation`, esql support it by applying bucket_sort
 - ES SQL API does not support `HAVING aggregation` that not show up in `SELECT`, esql support it
+- to use regex query, the column should be `keyword` type, otherwise the regex is applied to all the terms produced by tokenizer from the original text rather than the original text
```

Comment:
I am confused on what this mean? I supposed keyword type should always be used in for exact match, but text could be used to use regex

Reply:
From the ES doc and my experiment, if field is in 'text' format, tokenizer will break the original text into space separated words. In this case, regex is applied to each of these words, and if any one matches, the corresponding row will be returned.

For example, if "aaa bbb" is in 'text' format, it will be broken into "aaa" "bbb", and regex '[  ab]{7}' won't match, but regex 'a{3}' and 'b{3}' will both match. But if it is in 'keyword' format, regex  '[  ab]{7}' matches. I think the latter behavior is what we want.